src/Parser.js

```diff
@@ -0,0 +1,58 @@
+
```

Comment:
Better approach:
parse the xml once and store codes and publish details in dictionaries (avoid to scan the dom on each get method)